Sources/Demo/ResultDetailViewController.swift

```diff
@@ -0,0 +1,158 @@
+class ResultDetailViewController: UIViewController {
+    private var tableView = UITableView()
+    private var mapView: MapView
+
+    var result: SearchResult
+    private var resultComponents: [(name: String, value: String)] = []
+
+    init(result: SearchResult) {
+        self.result = result
+        self.resultComponents = result.toComponents()
+
+        let inset: CGFloat = 8
+        let padding = UIEdgeInsets(top: inset, left: inset, bottom: inset, right: inset)
+        self.mapView = MapView(
+            frame: .zero,
+            mapInitOptions: MapInitOptions(cameraOptions: CameraOptions(
+                center: result.coordinate,
+                padding: padding,
+                zoom: 15.5
+            ))
+        )
+
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("\(#function) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.dataSource = self
+        tableView.allowsSelection = false
+
+        for child in [tableView, mapView] {
+            child.translatesAutoresizingMaskIntoConstraints = false
+            view.addSubview(child)
+        }
+
+        NSLayoutConstraint.activate([
+            mapView.topAnchor.constraint(equalTo: view.topAnchor),
+            mapView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            mapView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            mapView.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 0.4),
+
+            tableView.topAnchor.constraint(equalTo: mapView.bottomAnchor),
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+        ])
+
+        /// Add annotations and set camera
+        let annotationsManager = mapView.annotations.makePointAnnotationManager()
+        annotationsManager.annotations = [result].map { result in
+            var point = PointAnnotation(coordinate: result.coordinate)
+            point.textField = result.name
+            UIImage(named: "pin").map { point.image = .init(image: $0, name: "pin") }
+            return point
+        }
+
+        if let annotation = annotationsManager.annotations.first {
+            mapView.camera.fly(
+                to: .init(center: annotation.point.coordinates, zoom: 15),
+                duration: 0.25,
+                completion: nil
+            )
+        }
+    }
+}
+
+extension ResultDetailViewController: UITableViewDataSource {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        2
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch section {
+        case 0:
+            resultComponents.count
+        default:
+            0
```

Comment:
Is a second section then necessary if it shows 0 rows?

Reply:
Ah yes I had an idea to add a second section but never followed-through. Maybe later on!